client/my-sites/backup/status/hooks.js

```diff
@@ -40,13 +40,13 @@ const useBackupDeltas = ( siteId, { before, after, number = 1000 } = {}, enabled
 
 	const isValidRequest = filter.before && filter.after;
 
-	const { data, isLoading } = useRewindableActivityLogQuery( siteId, filter, {
+	const { data, isInitialLoading } = useRewindableActivityLogQuery( siteId, filter, {
 		enabled: isValidRequest && enabled,
 		refetchOnWindowFocus: false,
 	} );
 
 	return {
-		isLoading,
+		isLoading: isInitialLoading,
```

Comment:
I think it can be helpful to rename the property returned by the hook as well, so it reflects the actual behavior and matches the underlying React Query API that we utilize. Otherwise, it might be confusing because the hook consumer could expect it to use `isLoading` under the hood and that wouldn't be true.

Reply:
I second what Marin said, otherwise I think this looks good.

Reply:
Makes sense - improved in a1d3d16cb8.

Thanks!